### ISSUE TITLE
Allow separating reports by subreddit/channel

### ISSUE THREAD

**Issue by Raia** [pokemontrades/porygon-bot#50]
Assistant mods aren't being modded on SVExchange, so we need to be able to exclude its reports from the assistant mod channel.

**Comment by not-an-aardvark** [comment]
Will we want to do the same thing with usernotes?

**Comment by YaManicKill** [comment]
Was this ever done?

**Comment by Raia** [comment]
No, we ended up just using multiple copies of the report script, one for each sub.
